docs/docs/development/localization.md

```diff
@@ -35,9 +35,9 @@ You'll see a list of languages ready for translation.
 
 #### Administrators can pull specific languages their Transifex account is associated with
 
-In BigBlueButton 2.2, 2.3, 2.4, 2.5 the script used for pulling the locales is located in the `bigbluebutton-html5` directory
+Since BigBlueButton 2.2 the script used for pulling the locales is located in the `bigbluebutton-html5` directory
 
-https://github.com/bigbluebutton/bigbluebutton/blob/v2.4.x-release/bigbluebutton-html5/transifex.sh
+https://github.com/bigbluebutton/bigbluebutton/blob/develop/bigbluebutton-html5/transifex.sh
```

Comment:
```suggestion
https://github.com/bigbluebutton/bigbluebutton/blob/v2.5.x-release/bigbluebutton-html5/transifex.sh
```

Starting with version 2.6, BigBlueButton integrates directly with Transifex, and pull requests are automatically received whenever a local reaches 100% translation completion on Transifex.